examples/mediapipe/customMediaPipe/index.html

```diff
@@ -0,0 +1,227 @@
+<html>
+<head>
+	<style>
+        body{
+            margin: 0px;
+        }
+        .headline {
+            text-align: center;
+        }
+
+        vwc-top-app-bar img {
+            margin-left: 16px
+        }
+
+        .maincard{
+            margin: 10 auto;
+            width: 1134px;
+            height: fit-content;
+            display: block;
+        }
+
+        .maincardlayout{
+            display: grid;
+            grid-template-columns: 454px 674px;
+            overflow: hidden;
+        }
+
+        .sourecevideo{
+            aspect-ratio: 1;
+            width: 100%;
+            background: white;
+            border-radius: 4px;
+            height: 100%;
+            object-fit: cover;
+            border: 1px solid var(--vvd-color-neutral-30);
+        }
+        
+        .previewvideodiv{
+            aspect-ratio: 1;
+            width: 100%;
+            background-image: url(images/vonage.png);
+            border-radius: 4px;
+            height: 100%;
+            object-fit: cover;
+            border: 1px solid var(--vvd-color-neutral-30);
+        }
+        .previewvideo{
+            width: 100%;
+            border-radius: 4px;
```

Comment:
it is not needed - because its div wrapper had `overflow:hidden`
```suggestion
            
```

Reply:
done